### PR TITLE
LPS-187960 Update usage of ModelListener.onAfterUpdate(T model) with ModelListener.onAfterUpdate(T originalModel, T model)

### PR DESCRIPTION
**Pull request by TamyrisTorres**
Ticket: https://liferay.atlassian.net/browse/LPS-187960

### CODE REVIEW

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Comment by TamyrisTorres** [comment]
ci:test:relevant

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Comment by TamyrisTorres** [comment]
ci:test:relevant

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaOnAfterUpdateParameterCheck.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+	@Override
+	protected String doProcess(
+			String fileName, String absolutePath, JavaTerm javaTerm,
+			String fileContent)
+		throws Exception {
+
+		JavaClass javaClass = (JavaClass)javaTerm;
+
+		if (!_checkMethods(javaClass)) {
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaOnAfterUpdateParameterCheck.java:41]: @TamyrisTorres, let's rename this method to `extendsFromBaseModelListener`

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaOnAfterUpdateParameterCheck.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+			return fileContent;
+		}
+
+		String newContent = javaTerm.getContent();
+
+		for (JavaTerm childJavaTerms : javaClass.getChildJavaTerms()) {
+			String javaMethodContent = childJavaTerms.getContent();
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaOnAfterUpdateParameterCheck.java:48]: @TamyrisTorres, let's rename `javaMethodContent` to `content` only to keep consistency

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaOnAfterUpdateParameterCheck.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+	private String _format(String methodCall, String newContent) {
+		List<String> parameterList = JavaSourceUtil.getParameterList(
+			methodCall);
+
+		if (parameterList.size() != 1) {
+			return newContent;
+		}
+
+		String[] split = StringUtil.split(parameterList.get(0), " ");
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaOnAfterUpdateParameterCheck.java:87]: @TamyrisTorres, how about we clone the first parameter and adjust it according to what we discussed offline?

File: modules/util/source-formatter/src/main/resources/sourcechecks.xml [hunk 4]
```diff
@@ -1601,6 +1601,10 @@
 			<category name="Upgrade" />
 			<description name="Replaces the references of the MultiVMPoolUtil class and also its methods usages." />
 		</check>
+		<check name="UpgradeJavaOnAfterUpdateParameterCheck">
+			<category name="Upgrade" />
+			<description name="Fill in the new parameter of the method 'onAfterUpdate' of ModelList class" />
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/resources/sourcechecks.xml:1606]: @TamyrisTorres, instead of `ModelList` please use `BaseModelListener`

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Pull request closed without merging**